integration_tests/e2e/application-type.cy.ts

```diff
@@ -0,0 +1,60 @@
+context('Application Type Page', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.signIn()
+    cy.visit('/log/application-type')
+  })
+
+  it('should display the page title', () => {
```

Comment:
I think you can just use https://github.com/ministryofjustice/hmpps-managing-prisoner-apps-staff-ui/blob/166d7dc6a267dd396a8af2e4ddbaf4bb11aaf1e3/integration_tests/pages/page.ts#L12 to do this step